pkgs/misc/uxn/default.nix

```diff
@@ -0,0 +1,50 @@
+{ pkgs ? import <nixpkgs> {} }:
+
+with pkgs;
+
+stdenv.mkDerivation rec {
+  pname = "uxn";
+  version = "2021-08-26-c6c31aa";
+
+  src = fetchFromSourcehut {
+    owner = "~rabbits";
+    repo = pname;
+    rev = "c6c31aa81546ac2348efd4031e3130ca22e9d9a0";
+    sha256 = "0r3qpx4cwbd9plp4y0yk14ys3impm31d47sa93d4n7vr4j7p7aim";
+  };
+
+  buildInputs = [ SDL2 ];
+
+  buildPhase = ''
```

Comment:
Yeah, it's not super elegant but at least there's not too much going on.

There is a `build.sh` in the repo but it does a bunch of things I don't think we should be doing, such as copying binaries to the home directory and even launching the emulator itself. The latter does end up failing because the environment doesn't match expectations (`$DISPLAY` isn't set or something like that) but all in all it felt less hacky performing a few manual steps than trying to work with the script.